data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/unapproachable_east/una_abilities.lst

```diff
@@ -39,7 +39,8 @@ Runescarring										CATEGORY:Special Ability	TYPE:SpecialQuality.Extraordinary
-Ritual Scarring		KEY:Runescarred Berserker ~ Ritual Scarring	CATEGORY:Special Ability	TYPE:Runescarred Berserker Class Feature.SpecialQuality.Extraordinary																	DESC:Through frequent disfiguration of her own skin, a runescarred berserker gains a +1 natural armor bonus to AC at 3rd level. This increases to +2 at 6th level, and +3 at 9th level.																																																																																																																																																																																																																																																																																																																																																																																					SOURCEPAGE:p.32
-Spawn of the Frost	KEY:Runescarred Berserker ~ Spawn of the Frost	CATEGORY:Special Ability	TYPE:Runescarred Berserker Class Feature.SpecialQuality.Supernatural																	DESC:Inured to the bitter cold of Rashemen's winters, a runescarred berserker gains cold resistance 5.																																																																																																																																																																																																																																																																																																																																																																																																		SOURCEPAGE:p.32
+Ritual Scarring		KEY:Runescarred Berserker ~ Ritual Scarring	CATEGORY:Special Ability	TYPE:Runescarred Berserker Class Feature.SpecialQuality.Extraordinary																	DESC:Through frequent disfiguration of her own skin, a runescarred berserker gains a +1 natural armor bonus to AC at 3rd level. This increases to +2 at 6th level, and +3 at 9th level.																																																																																																																																																																																																																																																																																																																																																																																					DEFINE:RunescarNaturalAC|0	BONUS:VAR|RunescarNaturalAC|1|PREVARGTEQ:RunescarredBerserkerLVL,3	BONUS:VAR|RunescarNaturalAC|1|PREVARGTEQ:RunescarredBerserkerLVL,6	BONUS:VAR|RunescarNaturalAC|1|PREVARGTEQ:RunescarredBerserkerLVL,9	BONUS:COMBAT|AC|RunescarNaturalAC|TYPE=NaturalArmor	SOURCEPAGE:p.32
```

Comment:
Why separate statements? This can be simplified with multiple if statements inside or using the native comparison.